### ISSUE TITLE
Mouse lock nor ESC does not seem to work

### ISSUE THREAD

**Issue by mikroskeem** [thesauri/jsmpeg-vnc-linux#6]
Hey, I got your e-mail about this project. Amazing to see you finally blew dust off from this project finally :smile: 

However, here are few issues I discovered:
1) Mouse lock does not work - mouse locks but does not really send anything to jsmpeg server. Without mouse lock, mouse events are reported just fine.
2) Escape button does not work. Must use `DISPLAY=:1 xdotool key Escape` for that

**Comment by thesauri** [comment]
Glad someone has use of it and thanks for reporting the issues! 😁

Seems like I never went about to porting mouse lock movement...
<img width="677" alt="Screen Shot 2020-12-25 at 22 37 28" src="https://user-images.githubusercontent.com/14876246/103142338-caecfb80-4701-11eb-8067-101bd02da017.png">

I'll have a look at this later™, but for now I tried to fix the Escape issue in #7. Could you verify that it works?

**Comment by mikroskeem** [comment]
Sadly that didn't seem to work. I've confirmed that Escape key is being sent by adding console.log call to sendKey function.

**Comment by thesauri** [comment]
Hmm. Did you try the `escape` branch? Haven’t merged the PR to master yet.

**Comment by mikroskeem** [comment]
Yep, I did. Did even a clean build.

Here's my testing setup:
```
Xvfb :1 -screen 0 1280x720x24
DISPLAY=:1 ../build/jsmpeg-vnc "desktop"
DISPLAY=:1 xev
```

Escape events never reach to `xev`

**Comment by thesauri** [comment]
Huh, interesting. I tried running `xev` and pressing escape yielded the following:

<img width="516" alt="Screen Shot 2020-12-26 at 19 28 12" src="https://user-images.githubusercontent.com/14876246/103158071-24b0fc80-47ba-11eb-9f12-7b3d72a9fb2b.png">

Same result in both Safari and Firefox on macOS. I also tried running Firefox in a container and I was able to dismiss focus by pressing escape.

Regarding mouse lock, I implemented relative mouse movement in #7 so that should work now.

**Comment by mikroskeem** [comment]
Disregard my report about Esc not working - I accidentally did `../bin/jsmpeg-vnc` instead of `../build/jsmpeg-vnc`, running old build instead.

Everything seems to work perfectly

Can continue building my tiny wintry project
![dosbox](https://user-images.githubusercontent.com/3490861/103162548-a163cb00-47fa-11eb-85af-5f595f8fb12a.png)
